tests/components/laundrify/test_coordinator.py

```diff
@@ -0,0 +1,50 @@
+"""Test the laundrify coordinator."""
+
+from laundrify_aio import exceptions
+
+from homeassistant.components.laundrify.const import DOMAIN
+from homeassistant.core import HomeAssistant
+
+from . import create_entry
+
+
+async def test_coordinator_update_success(hass: HomeAssistant):
+    """Test the coordinator update is performed successfully."""
+    config_entry = create_entry(hass)
+    await hass.config_entries.async_setup(config_entry.entry_id)
+    await hass.async_block_till_done()
+    coordinator = hass.data[DOMAIN][config_entry.entry_id]["coordinator"]
```

Comment:
Please don't access integration details like hass.data or the coordinator in the tests.

https://developers.home-assistant.io/docs/development_testing#writing-tests-for-integrations